Gems/AtomLyIntegration/CommonFeatures/Code/Source/Debug/RenderDebugEditorComponent.cpp

```diff
@@ -25,7 +25,7 @@ namespace AZ::Render
                 editContext->Class<RenderDebugEditorComponent>(
                     "DebugRendering", "Controls for debugging rendering.")
                     ->ClassElement(Edit::ClassElements::EditorData, "")
-                        ->Attribute(Edit::Attributes::Category, "Atom")
+                        ->Attribute(Edit::Attributes::Category, "Graphics - Debugging")
```

Comment:
update: It looks like this should be `Graphics/Debugging` now?

Reply:
Nice catch! Fixed it.

Reply:
Amazing thank you, does look way better! 🙂